src/webgpu/api/validation/encoding/cmds/index_access.spec.ts

```diff
@@ -115,17 +124,22 @@ class F extends ValidationTest {
     pass.drawIndexedIndirect(indirectBuffer, indirectOffset);
     pass.endPass();
 
-    this.device.queue.submit([encoder.finish()]);
+    if (isSuccess) {
```

Comment:
The WebGPU specification doesn't require an error to be produced for `drawIndexedIndirect` because we don't know on the CPU how many indices will be used. I think we need to keep the TODO to check robust behavior for the indirect case. (note that for OT Chromium disables `drawIndecedIndirect` because it is missing the validation).

Reply:
Done.